public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts

```diff
@@ -94,6 +104,14 @@ export const Messages = {
         username: 'Your database user name',
         password: 'Your database password',
       },
+      postgresqlDetails: {
+        address: 'Public DNS hostname of your instance',
```

Comment:
Here the same logic applies, these strings are already presented in `mainDetails`

Reply:
thx